### ISSUE TITLE
export clearFields action creator

### ISSUE THREAD

**Issue by juicylevel** [erikras/redux-form#3590]
Hello! 
You forgot to add the export clearFields action creator in the index.js 
The absence of export causes an error: `Uncaught TypeError: (0 , _reduxForm.clearFields) is not a function`

**Comment by juicylevel** [comment]
`npm install redux-form --save`
package.json - version 7.1.2

clearFields action does not exist in the source code of the redux-form module

**Comment by gustavohenke** [comment]
It wasn't released yet.

**Comment by thomaspersson** [comment]
So when will clearFields be released?

**Comment by mk0x9** [comment]
But it's in the docs!

https://redux-form.com/7.1.2/docs/api/actioncreators.md/#-code-clearfields-form-string-keeptouched-boolean-persistentsubmiterrors-boolean-fields-string-code-

**Comment by gustavohenke** [comment]
@erikras anything strange happened the last time you published the docs? 😄

**Comment by caseybaggz** [comment]
Any update on this?

**Comment by KrzysztofKarol** [comment]
If someone needs that @Wikom has already a solution:
https://github.com/Wikom/wikom-form/blob/a7da36ddc2c39644d719c04b862e16a0d17c5034/src/actions/clearFields.js

Just modified it a little for me
```javascript
// TODO: Use redux-form's clearFields https://github.com/erikras/redux-form/issues/3590
import { change } from 'redux-form';

const clearFields = (form, fields) => dispatch =>
  fields.forEach(field => dispatch(change(form, field, null)));

export { clearFields };
```

**Comment by caseybaggz** [comment]
This was fixed with #3626

**Comment by KrzysztofKarol** [comment]
Yes, I know it. But is not in latest npm package 7.1.2.

On Nov 27, 2017 15:43, "Casey Baggz" <notifications@github.com> wrote:

This was fixed with #3626 <https://github.com/erikras/redux-form/pull/3626>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/erikras/redux-form/issues/3590#issuecomment-347202317>,
or mute the thread

**Comment by gustavohenke** [comment]
@KrzysztofKarol the latest is v7.2.0 now :)
